sound/soc/codecs/rt712-sdca-sdw.c

```diff
@@ -178,13 +194,15 @@ static int rt712_sdca_read_prop(struct sdw_slave *slave)
 	unsigned long addr;
 	struct sdw_dpn_prop *dpn;
 
+	sdw_slave_read_prop(slave);
+
 	prop->scp_int1_mask = SDW_SCP_INT1_BUS_CLASH | SDW_SCP_INT1_PARITY;
 	prop->quirks = SDW_SLAVE_QUIRKS_INVALID_INITIAL_PARITY;
 
 	prop->paging_support = true;
 
 	/* first we need to allocate memory for set bits in port lists */
-	prop->source_ports = BIT(4); /* BITMAP: 00010000 */
+	prop->source_ports = BIT(8) | BIT(4); /* BITMAP: 100010000 */
```

Comment:
I am really confused, is the port8 supported on all versions?

Reply:
The port8 supports on the VB. 
The DMIC function will use port8 and the driver only creates the "rt712-sdca-aif3" of DMIC DAI when using the VB.
It is easy to add port8 here and allocate memory for set bits in port lists.
The VA could not use port8 because it would not create the DMIC DAI.